src/okd/build_images.sh

```diff
@@ -4,14 +4,23 @@ set -euo pipefail
 export LC_ALL=C.UTF-8
 export LANG=C.UTF-8
 
-TARGET_REGISTRY=${TARGET_REGISTRY:-ghcr.io/microshift-io/okd}
+# Production registry - must be provided via TARGET_REGISTRY environment variable
+# or defaults to the upstream registry if not specified
+PRODUCTION_REGISTRY="${TARGET_REGISTRY:-ghcr.io/microshift-io/okd}"
+# Automatically derive staging registry by appending '/okd-staging' subpath
+STAGING_REGISTRY="${PRODUCTION_REGISTRY}/okd-staging"
```

Comment:
Let's set the staging registry path without the `/okd` suffix.
Something like the following:

```suggestion
STAGING_REGISTRY="$(dirname ${PRODUCTION_REGISTRY})/okd-staging"
```

Reply:
Done